spec/team_result_parser_spec.rb

```diff
@@ -0,0 +1,12 @@
+require './lib/team_result'
+require './lib/team_result_parser'
+
+RSpec.describe TeamResultParser do
+  it "Init and get info" do
```

Comment:
Probably want to refactor so that we have describe above the it block and then following pattern of initialize in the first it block and attributes in the second.